akka-projection-grpc/src/main/scala/akka/projection/grpc/producer/scaladsl/EventProducer.scala

```diff
@@ -84,17 +87,30 @@ object EventProducer {
 
   /**
    * The gRPC route that can be included in an Akka HTTP server.
+   *
+   * @param sources All sources that should be available from this event producer
    */
   def grpcServiceHandler(sources: Set[EventProducerSource])(
       implicit system: ActorSystem[_]): PartialFunction[HttpRequest, scala.concurrent.Future[HttpResponse]] = {
 
+    grpcServiceHandler(sources, None)
+  }
+
+  /**
+   * The gRPC route that can be included in an Akka HTTP server.
+   *
+   * @param sources All sources that should be available from this event producer
+   * @param interceptor An optional request interceptor applied to each request to the service
+   */
+  @ApiMayChange
```

Comment:
should we add this everywhere?

Reply:
All over the grpc APIs? Yes, maybe a good idea to leave some changing room here. I'll do that.

Reply:
yes, might be best